TLM/TMPE.API/Notifier.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+
+namespace TrafficManager.API
+{
+    public class OnModifiedEventArgs
+    {
+        /// <summary>
+        /// node/segment (citizen/vehicle in future) that has been modified.
+        /// </summary>
+        public InstanceID InstanceID;
+
+        /// <summary>
+        /// Reserved: the sender that caused the rule to be modified.
+        /// </summary>
+        public object Sender;
+
+        /// <summary>
+        /// Reserved: The rule that has been modified. object type is decided by sender.
+        /// </summary>
+        public object Data;
+    }
+
+    public static class Notifier
+    {
+        public static event Action EventLevelLaoded;
```

Comment:
Typo: EventLevelL **ao** ded.
Name is not the best => it's already an **event**, don't need to name it.